front/components/agent_builder/AgentBuilderLeftPanel.tsx

```diff
@@ -58,19 +59,18 @@ export function AgentBuilderLeftPanel({
           />
         }
       />
-      <div className="flex-1 overflow-y-auto">
+      <ScrollArea className="flex-1">
         <div className="space-y-4 p-4">
           <AgentBuilderInstructionsBlock
             agentConfigurationId={agentConfigurationId}
           />
-          <Separator />
```

Comment:
@pinotalexandre there was no separator between instructions and capabilities block on figma so I removed it, I guess it's intentional? (I think we can even remove it the other one too)